### DISCUSSION TITLE
Fixed api_wrapper.py

### DISCUSSION THREAD

**Discussion by wacns** [thejoabo/virtualfisher-bot#141] [Show and tell]
Hello @thejoabo , I've solved the problem regarding to the new discord API endpoints' update. Have a look on it and feel free to discuss anything that shows up in your mind and merge the PR #140 if everything is fine.

**Comment by thejoabo** [comment]
Hello @wacns, thank you very much for your contribution. I decided yesterday to develop this entire bot from scratch once and for all. I'm migrating to Typescript and changing to an observer pattern to allow plugins, making it way easier for people to contribute and expand. Currently, I'm working on the core functionalities and experimenting with new approaches. Feel free to email me or send me your Discord username there if you want to discuss something else.
